src/main/java/com/puppycrawl/tools/checkstyle/site/PropertiesMacro.java

```diff
@@ -62,7 +63,10 @@ public class PropertiesMacro extends AbstractMacro {
     /** A newline with 20 spaces of indentation. */
     private static final String INDENT_LEVEL_20 = SiteUtil.getNewlineAndIndentSpaces(20);
 
-    /** The name of the current module being processed. */
+    /** The name of the token property being processed. */
+    private static final String TOKENS_PROPERTY = "tokens";
```

Comment:
Hi @AayushSaini101, i have also been working on some migration ticket. I am curious to know if this field is being used anywhere. And, what was the error what forced you to add this field, if any? Your insights on the same can be helpful for me. Thanks.

Reply:
@relentless-pursuit Actually they are using a particular arrangement of the properties. All the properties in the sorted order instead of the tokens, that's why we need to add tokens at the end.